reference/checkouts.v3.yml

```diff
@@ -922,39 +905,30 @@ definitions:
             type: integer
             format: int32
           email:
-            description: >-
-              The cart's email. This is the same email that is used in the
-              billing address
+            description: The cart's email. This is the same email that is used in the billing address
             type: string
           currency:
             title: Currency
             type: object
             properties:
               code:
-                description: >-
-                  ISO-4217 currency code. (See:
-                  http://en.wikipedia.org/wiki/ISO_4217.)
+                description: 'ISO-4217 currency code. (See: http://en.wikipedia.org/wiki/ISO_4217.)'
                 type: string
           base_amount:
             description: 'Cost of cart''s contents, before applying discounts.'
             type: number
-            format: double
+          channel_id:
+            description: ID of channel
+            type: number  
           discount_amount:
             description: Discounted amount.
-            type: number
-            format: double
+            type: number            
           cart_amount_inc_tax:
-            description: >-
-              Sum of line-items amounts, minus cart-level discounts and coupons
-              including tax
+            description: 'Sum of line-items amounts, minus cart-level discounts and coupons including tax'
             type: number
-            format: double
```

Comment:
Looks like this property supports format: double. For example, "cart_amount_inc_tax": 209.99999999999997.